test/unit/__snapshots__/Deposits.spec.ts.snap

```diff
@@ -1,5 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`unit/Deposits #onERC721Received on successful transfer with staking data has gas cost 1`] = `211717`;
+exports[`unit/Deposits #onERC721Received on successful transfer with staking data has gas cost 1`] = `239145`;
```

Comment:
this cost is bigger than the savings on unstake

Reply:
see above about making it one slot